src/argument_parsers/copy_argument_parser.cpp

```diff
@@ -0,0 +1,18 @@
+
+#include <filesystem>
+#include <string>
+#include <argument_parsers/copy_argument_parser.h>
+#include <iostream>
+
+CopyArgumentParser::CopyArgumentParser(const std::string& source, const std::string& destination){
+    _source = std::filesystem::path{ source };
```

Comment:
please, use the initialization list for member initialization
https://en.cppreference.com/w/cpp/language/constructor